### PR TITLE
Terraform aws provider v4

### PR DESCRIPTION
**Pull request by mateuszmidor**
KNTK-396

Update AWS Terraform module:
- use AWS provider version >= 4.0.0, < 5.0.0
- use kentik-cloudexport version >=0.4.0, < 1.0.0
- use Terraform version >= 1.0.0, < 2.0.0

### CODE REVIEW

**Review by danielfurman — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]

**Comment by mmac-m3a** [comment]
I'll wait for Daniel's approval.

**Review by danielfurman — APPROVED** [review]

**Review by mmac-m3a — APPROVED** [review]

**Pull request merged**
